oci/charmed-opensearch/image.yaml

```diff
@@ -1,21 +1,21 @@
 version: 1
 
-#release:
-#  2:
-#    end-of-life: "2024-05-01T00:00:00Z"
-#    candidate: 2.8.0-22.04_candidate
-#    edge: 2-edge
+release:
```

Comment:
these will fail for 2 reasons:
 1. 2-edge is not a proper channel. It would be `2_edge`
 2. `2.8.0-22.04_candidate` doesn't exist yet, so you'd be pointing 2_candidate to `null`

Can delete this section for your first PR